### PR TITLE
Remove cutes dependency and obsolete android controls

### PR DESCRIPTION
**Pull request by pvuorela**
@rainemak @jpetrell

### CODE REVIEW

File: plugin/utiltools.cpp [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+void UtilTools::cleanRpmDb(QJSValue successCallback, QJSValue errorCallback)
+{
+    execute(SystemTool, QStringList("repair_rpm_db"), successCallback, errorCallback);
+}
+
+void UtilTools::restartKeyboard(QJSValue successCallback, QJSValue errorCallback)
+{
+    QStringList arguments;
+    arguments << "--user" << "restart" << "maliit-server.service";
```
Comment (rainemak) [inline plugin/utiltools.cpp]: .service suffix is not necessary

File: plugin/utiltools.cpp [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    if (status == QProcess::NormalExit && exitCode == 0) {
+        if (callbacks.first.isCallable()) {
+            QJSValue result = callbacks.first.call();
+            if (result.isError()) {
+                qmlInfo(this) << "Error executing callback";
+            }
+        }
+    } else {
+        if (callbacks.second.isCallable()) {
```
Comment (rainemak) [inline plugin/utiltools.cpp]: One level of indentations could be removed like:
```
} else if (callbacks.second.isCallable()) { 
    QJSValue result = callbacks.second.call();
    if (result.isError()) {
        qmlInfo(this) << "Error executing error callback";
    }
}
```

File: plugin/utiltools.cpp (hunk 1 above)
Comment (pvuorela) [inline plugin/utiltools.cpp]: Works without, but was already like that and as code I guess this doesn't need the abbreviated format as much as command line usage.

**Comment by rainemak** [comment]
Added some minor comments. LGTM

**Pull request merged**